React/Project structure.md

```diff
@@ -284,7 +284,60 @@ Example:
     └── WelcomeCard.elements.ts
 ```
 
-> Moving things to`.elements.tsx` should be the last step in the development process and it should only be used for organizational purposes, i.e. when the main component becomes cluttered and unreadable.
+In this example `WelcomeCardOverlay` is a pure Functional component because chakra factory doesn't allow custom `isOpen` prop.
```

Comment:
```suggestion
In this example `WelcomeCardOverlay` is a pure Functional component because chakra factory doesn't allow the custom `isOpen` prop.
```